core/trino-server-rpm/src/test/java/io/trino/server/rpm/TestDummy.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.server.rpm;
+
+import org.junit.jupiter.api.Test;
+
+public class TestDummy
```

Comment:
Why do we need to add this? It wasn't needed before

Reply:
I guess that the previous plugin was providing a test:test goal.